### ISSUE TITLE
Coinbase wallet: window.walletLinkExtension.setProvider info is not defined

### ISSUE THREAD

**Issue by everdimension** [walletlink/walletlink#100]
Hi!

### Problem
I have updated the package to `^2.1.0` and when I open my app in coinbase wallet I'm getting this error:
```
window.walletLinkExtension.setProvider info is not defined
```

I believe it's from this line: https://github.com/walletlink/walletlink/blob/d69239381ebe9a5ca30f0f04304c9adc0218ed8a/js/src/WalletLink.ts#L109

I tried inspecting the `walletLinkExtension` object and it seems to only have these keys:

```
_events,_eventsCount,_maxListeners,isCipher,isToshi,isCoinbaseWallet,_filterPolyfill,_subscriptionManager,appName,appVersion,_address,_chainId
```

### Coinbase wallet version:
`24.5.495` on iOS

**Comment by vishnumad** [comment]
Thanks for reporting! This should be fixed in `2.1.3`.

**Comment by gsteele178** [comment]
Dudes can you help me get my eth into my wallet?
